source/unifiedbuffer.ts

```diff
@@ -0,0 +1,142 @@
+
+/* spellchecker: disable */
+
+import { Buffer } from './buffer';
+import { Context } from './context';
+import { Initializable } from './initializable';
+
+/* spellchecker: enable */
+
+
+export class UnifiedBuffer extends Initializable {
+
+    protected _cpuBuffer: ArrayBuffer;
+    protected _gpuBuffer: Buffer;
+    protected _updates = new Array<Update>();
+
+    usage: GLenum;
```

Comment:
Please convert this member to a protected member and add getter and setter accordingly.